include/usb_cam/formats/av_pixel_format_helper.hpp

```diff
@@ -958,6 +960,152 @@ inline AVPixelFormat get_av_pixel_format_from_string(const std::string & str)
   return STR_2_AVPIXFMT.find(fullFmtStr)->second;
 }
 
+/// @brief Get ROS PixelFormat from AVPixelFormat.
+/// @param avPixelFormat AVPixelFormat
+/// @return String specifying the ROS pixel format.
+inline std::string get_ros_pixel_format_from_av_format(const AVPixelFormat & avPixelFormat)
+{
+  std::string ros_format = "";
+
+  switch (avPixelFormat) {
+    default:
+      {
+        ros_format = usb_cam::constants::UNKNOWN;
+      }
+      break;
```

Comment:
@boitumeloruf I know its not necessary, but could you move the `default` cases for the switches here to be last? You could also remove the `break` then from each of the default cases